realm/src/main/java/io/realm/Realm.java

```diff
@@ -163,19 +163,18 @@ protected void finalize() throws Throwable {
     }
 
     /**
-     * Realm static constructor for the default Realm "default.realm".
-     * {@link #close()} must be called when you are done using the Realm instance.
-     * <p>
-     * It sets auto-refresh on if the current thread has a Looper, off otherwise.
-     *
+     * Realm static constructor for the default Realm file "default.realm".
```

Comment:
Gradle has nothing to do with it. It simply calls the JDK facilities to generate Javadocs.